crates/script/src/runner.rs

```diff
@@ -358,7 +358,7 @@ impl ScriptRunner {
         calldata: &Bytes,
         value: U256,
     ) -> Result<u64> {
-        let mut gas_used = res.gas_used;
+        let mut gas_used = res.gas_used + res.gas_refunded + res.stipend;
```

Comment:
we should also do here:

```
 let optimistic_gas_limit = (gas_used + gas_refund + CALL_STIPEND_GAS) * 64 / 63;
```

I wonder why this fn can end up returning too low gas in the first place

ref

https://github.com/ethereum/go-ethereum/blob/a5a4fa7032bb248f5a7c40f4e8df2b131c4186a4/eth/gasestimator/gasestimator.go#L135

and

https://github.com/paradigmxyz/reth/blob/339f18c48f829a2f2fca440fb255995e23c7161f/crates/rpc/rpc-eth-api/src/helpers/estimate.rs#L192-L198